generators/ruby/sdk/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## [0.0.6] - 2024-02-07
+- Improvement: license files are now specified within the gem config if they are provided
+- Improvement: we now create a Github workflow file for publishing the gem automatically, in the presence of a `publishInfo` configuration
```

Comment:
Users don't know about `publishInfo`, can we add an example of the `generators.yml` configuration with the `output` key

Reply:
Is this meant to be consumed by users?

Reply:
```yaml
generators: 
  - name: 
     output: 
       location: rubygems
       ...
```

Reply:
Yes it is